cmd/main.go

```diff
@@ -14,7 +14,7 @@ func main() {
 	storage := file_storage.NewFileStorage("./reddit_jokes.json")
 	template := views.NewTemptale("./templates/")
 	server := http.Server{
-		Addr:              ":8000",
+		Addr:              ":8443",
```

Comment:
no, the server will work with an unencrypted connection. He doesn't know that there's encryption between the nginx and a client.